scribejava-apis/src/main/java/com/github/scribejava/apis/GitLabApi.java

```diff
@@ -0,0 +1,56 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.extractors.OAuth2AccessTokenJsonExtractor;
+import com.github.scribejava.core.extractors.TokenExtractor;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.utils.OAuthEncoder;
+
+public class GitLabApi extends DefaultApi20 {
+
+    private static final String ACCESS_TOKEN_ENDPOINT_PATH = "/oauth/token";
+    private static final String AUTHORIZATION_URL =
+            "%s/oauth/authorize?client_id=%s&redirect_uri=%s&response_type=code";
+    private static final String DEFAULT_BASE_URL = "https://gitlab.com";
+
+    private final String baseUrl;
+
+    public GitLabApi(String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    protected GitLabApi() {
+        this(DEFAULT_BASE_URL);
+    }
+
+    private static class InstanceHolder {
+        private static final GitLabApi INSTANCE = new GitLabApi();
+    }
+
+    public static GitLabApi instance() {
+        return InstanceHolder.INSTANCE;
+    }
+
+    @Override
+    public TokenExtractor<OAuth2AccessToken> getAccessTokenExtractor() {
+        return OAuth2AccessTokenJsonExtractor.instance();
+    }
```

Comment:
it's not needed anymore. It's default extractor now, just rebase to the master, pls